coffee_cmd/src/main.rs

```diff
@@ -42,6 +45,21 @@ async fn main() -> Result<(), CoffeeError> {
             // and the coffee script
             coffee.setup(&cln_conf).await
         }
+        CoffeeCommand::Show { plugin } => match coffee.show(&plugin).await {
+            Ok(val) => {
+                let mut skin = MadSkin::default();
+                skin.table.align = Alignment::Center;
+                skin.set_headers_fg(AnsiValue(178));
+                skin.bold.set_fg(Yellow);
+                skin.italic.set_fg(Magenta);
+                skin.scrollbar.thumb.set_fg(AnsiValue(178));
+                skin.code_block.align = Alignment::Center;
+
+                skin.print_text(val["show"].as_str().unwrap());
+                Ok(())
+            }
+            Err(err) => Err(err),
```

Comment:
This is nice, thank you!

I was wondering if we can do a little bit better and allow to scroll the terminal and not just print all the content?

Here is an example https://github.com/Canop/termimad/blob/main/examples/scrollable/main.rs